android/src/main/java/com/magicleap/magicscript/scene/nodes/video/UiSubtitlesNode.kt

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright (c) 2019-2020 Magic Leap, Inc. All Rights Reserved
```

Comment:
if this is a new file then it shouldn't have 2019 since it was created in 2020.

Reply:
Done